checkout-ui-custom/src/_js/_v.custom.checkout.ui.js

```diff
@@ -47,28 +47,22 @@ class checkoutCustom {
   }
 
   checkForFreightSimulation(func) {
-    // Wait until it have the vtex runtime to call the functions
-    const checkRuntime = setInterval(function() {
+    const _this = this
+
+    const isRenderRuntime =
+      window.vtex !== undefined && window.vtex.renderRuntime !== undefined
+
+    _this.awaitForElement(isRenderRuntime, 20, () => {
```

Comment:
I would create a const in order to make the code clearer:
Example
```ts
const MAX_SET_INTERNAVAL_LOOP = 20
```

```suggestion
   _this.awaitForElement(isRenderRuntime, MAX_SET_INTERNAVAL_LOOP, () => {
```

Reply:
But each element would need a different max, or do you mean, set a const before each call `awaitForElement` function?

Reply:
yes, absolutely, only to turn it clearer instead of using literals as parameters.